.changeset/twenty-bananas-relate.md

```diff
@@ -0,0 +1,9 @@
+---
+'@vanilla-extract/css': patch
+'@vanilla-extract/integration': patch
+'@vanilla-extract/private': patch
+'@vanilla-extract/recipes': patch
+'@vanilla-extract/sprinkles': patch
+---
+
+Export types with `export { type T }` syntax
```

Comment:
[`rollup-plugin-dts@6`](https://github.com/Swatinem/rollup-plugin-dts/blob/388052e8cc20022357efb83d968ea9d84b41e22d/CHANGELOG.md#600) started emitting type exports this way. These are the only packages that export types.